### PR TITLE
Fixed formating errors

### PR DESCRIPTION
**Pull request by milanperera**

### CODE REVIEW

File: modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java [hunk 1]
```diff
@@ -191,99 +191,96 @@ private void startUnRegistration() {
...
-			if (Constants.DEBUG_MODE_ENABLED) {
-				Log.d(TAG, "Calling onResume");
-			}
-
-			String regFlag = Preference.getString(context, getResources().getString(R.string.shared_pref_registered));
-
-			if(getResources().getString(R.string.shared_pref_reg_success).equals(regFlag)) {
-
-				if (CommonUtils.isNetworkAvailable(context)) {
+		if(getResources().getString(R.string.shared_pref_reg_success).equals(regFlag)) {
```
Comment (DilanUA) [inline modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java]: improper spacing

File: modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java [hunk 2]
```diff
@@ -191,99 +191,96 @@ private void startUnRegistration() {
...
+			if (CommonUtils.isNetworkAvailable(context)) {
 
-					String serverIP =
-							Preference.getString(AlreadyRegisteredActivity.this,
+				String serverIP =
+						Preference.getString(AlreadyRegisteredActivity.this,
```
Comment (DilanUA) [inline modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java]: unnecessary spacing

File: modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java [hunk 3]
```diff
@@ -191,99 +191,96 @@ private void startUnRegistration() {
...
-							ServerConfig utils = new ServerConfig();
-							utils.setServerIP(serverIP);
-
-							CommonUtils.callSecuredAPI(AlreadyRegisteredActivity.this,
-									utils.getAPIServerURL() + Constants.IS_REGISTERED_ENDPOINT + regId,
-									HTTP_METHODS.GET,
-									null, AlreadyRegisteredActivity.this,
-									Constants.IS_REGISTERED_REQUEST_CODE);
-						}
+										.getString(R.string.shared_pref_ip)
```
Comment (DilanUA) [inline modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java]: better moving . sign upwards

File: modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java [hunk 4]
```diff
@@ -191,99 +191,96 @@ private void startUnRegistration() {
...
+						initiateUnregistration();
 					} else {
-						showUnregisterDialog();
+						ServerConfig utils = new ServerConfig();
+						utils.setServerIP(serverIP);
+							CommonUtils.callSecuredAPI(AlreadyRegisteredActivity.this,
+								utils.getAPIServerURL() + Constants.IS_REGISTERED_ENDPOINT + regId,
+								HTTP_METHODS.GET,
+								null, AlreadyRegisteredActivity.this,
+								Constants.IS_REGISTERED_REQUEST_CODE);
```
Comment (DilanUA) [inline modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java]: unnecessary indentation

File: modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java (hunk 1 above)
Comment (milanperera) [inline modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java]: Fixed in [#70]

File: modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java (hunk 2 above)
Comment (milanperera) [inline modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java]: Fixed in [#70]

File: modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java (hunk 3 above)
Comment (milanperera) [inline modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java]: Fixed in [#70]

File: modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java (hunk 4 above)
Comment (milanperera) [inline modules/mobile-agents/android/client/src/org/wso2/mdm/agent/AlreadyRegisteredActivity.java]: Fixed in [#70]

**Pull request merged**